pkg/fanal/analyzer/language/java/jar/jar.go

```diff
@@ -49,18 +47,13 @@ func newJavaLibraryAnalyzer(options analyzer.AnalyzerOptions) (analyzer.PostAnal
 func (a *javaLibraryAnalyzer) PostAnalyze(ctx context.Context, input analyzer.PostAnalysisInput) (*analyzer.AnalysisResult, error) {
 	// TODO: think about the sonatype API and "--offline"
 	var err error
-	a.once.Do(func() {
```

Comment:
We use `PostAnalyze` for `jar` files.
We find all `jar` files => save in [our fs](https://github.com/aquasecurity/trivy/blob/562723f0a701f3aaf7d1aa16bdea490209af5bfa/pkg/mapfs/fs.go#L30) => run `PostAnalyze` function to parse all found `jar` files.

`PostAnalyze` is used once per run (in Client/Server mode - `Trivy` analyzes on the client side), so we can remove `once`.

Reply:
Makes sense